### ISSUE TITLE
[Bug] card due count doesn't match with Anki Desktop after sync

### ISSUE THREAD

**Issue by david-allison** [ankidroid/Anki-Android#9368]
This is with the "legacy timezone" code

See: https://forums.ankiweb.net/t/anki-droid-card-due-dont-match-with-anki-desktop/10955

This can be fixed by: "Anki Desktop - Preferences - Scheduling - Disable "legacy" timezone", but the code should match, and it doesn't currently

**Comment by david-allison-1** [comment]
Tried and can't repro this. Definitely a problem

**Comment by cafce25** [comment]
I have that problem, using UTC as my timezone everywhere, AnkiWeb and Desktop agree on the cards due and times, AnkiDroid disagrees. If anybody could point me to were this issue might be I can try to create a patch but I don't know the codebase at all.

**Comment by david-allison-1** [comment]
Perfect, thanks!

I believe it'll be a difference in the results of these variables on Android and Anki Desktop:

https://github.com/ankidroid/Anki-Android/blob/79981a009745916ce9fffeed9cee9bb74f4ee3e3/AnkiDroid/src/main/java/com/ichi2/libanki/sched/SchedV2.java#L2186-L2187

The Android values can be accessed via a debugger. 

The Anki Desktop code to obtain these values is:

* Open console (`Ctrl+Shift+;`)/(`Ctrl+Shift+Colon`)
```
pp(mw.col.sched.today)
pp(mw.col.sched.dayCutoff)
```

Latest Anki code is: https://github.com/ankitects/anki/blob/main/pylib/anki/scheduler/v2.py

**Comment by YoungMind1** [comment]
I have the same problem, my timezone is UTC +3:30 Tehran
this problem happened to me the day we changed the clocks(Daylight saving time thingy)
my android is on 20 59
while my ankiWeb and desktop are on 20 61

**Comment by aarjav24** [comment]
I'm facing a similar problem. The card count on AnkiDroid and Anki Desktop don't match for me until late in the evening when they finally show identical number of due cards. Until then, AnkiDroid says I have zero cards due and I'm unable to review any cards until the evening, which has led me to miss days.
![image](https://user-images.githubusercontent.com/68058926/135956943-9c8acba6-8ac0-4994-9a6e-223fc68d3835.png)
![Screenshot_20211005-074851_2](https://user-images.githubusercontent.com/68058926/135957219-a530f7ad-c785-4805-9fd5-b308ffc8bb64.jpg)

**Comment by david-allison-1** [comment]
Updated the OP with a solution for end-users. I can't reproduce this issue and the thread is designed to help find users who can debug and determine why it's occurring.

**Comment by mpPie** [comment]
I had this issue. I'll try to be as detailed as possible to reproduce this.

I started with a new account and fresh installation.

I created my custom note-type, deck and cards on Anki desktop.
I created an Anki account and syncronized the collection.
I opened Ankidroid and syncronized.
I changed the deck options from mobile and syncronized. I did my daily cards.
The next day I opened the app and found out my due count to 0. On desktop it was showing a correct due count.

I solved as other did, disabling "Legacy Timezone Handling" on desktop.
